tests/vmi_configuration_test.go

```diff
@@ -506,8 +503,11 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		Context("with BIOS bootloader method and no disk", func() {
 			It("[test_id:5265]should find no bootable device by default", func() {
 				By("Creating a VMI with no disk and an explicit network interface")
-				vmi := tests.NewRandomVMI()
-				tests.AddExplicitPodNetworkInterface(vmi)
+				vmi := libvmi.NewCirros(
```

Comment:
Please use `libvmi.New` as I find this confusing.

Reply:
[Done](https://github.com/kubevirt/kubevirt/compare/177fb3ffc5331fe90ecb2c39ce6bae29272950a8..30a985da41b8b304bdc668e44e3864b39783734e)